hooks/useEscrowedEXA.ts

```diff
@@ -43,10 +43,18 @@ export function useUpdateStreams() {
   const fetchStreams = useCallback(async () => {
     if (EXA && esEXA) {
       setLoading(true);
-      const data = await request<{ streams: Stream[] }>(
-        getStreams(EXA.address.toLowerCase(), walletAddress || zeroAddress, esEXA.address.toLowerCase(), false),
-        'sablier',
-      );
+
+      let data;
```

Comment:
Instead of using ley Instead od cost, you could wrap the test of the function  in the try block